src/components/graph-view.js

```diff
@@ -188,16 +196,19 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       .call(this.zoom);
 
     this.selectedView = d3.select(this.view);
-
+    if(this.props.initialBBox)
+      this.handleZoomToFitImpl(this.props.initialBBox, 0);
```

Comment:
Please use destructuring instead of always referencing `this.props.variable`. The first lines of a function should be:

```
function myFunc(){
  const { prop1, prop2 } = this.props;
  ...
}
```